.gitignore

```diff
@@ -11,3 +11,4 @@ build/
 composer.phar
 .idea/
 vendor/
+public/js/CDash_*.min.js*
```

Comment:
> Aside: Some schools of thought would check in this file and only have developers run gulp. There are advantages and disadvantages either way.

What do you think about doing this only on the release branch? I fear that this would be too noisy for master.  The .js filename will change every commit, and thus all the .html files will be modified too.